contracts/ClaimsManager.sol

```diff
@@ -51,42 +51,33 @@ contract ClaimsManager is
         public
```

Comment:
Small update for consistency: "admin" means manager or account with the admin role. An admin can do everything that a manager can do. An admin can do everything that any other role can do. In practice, the manager will likely not give this admin role to anyone else so this is not very consequential, just future-proofs things in case we want to implement a limited admin contract in the future.